django/db/backends/mysql/client.py

```diff
@@ -11,7 +11,7 @@ def settings_to_cmd_args(cls, settings_dict, parameters):
         args = [cls.executable_name]
         db = settings_dict['OPTIONS'].get('db', settings_dict['NAME'])
         user = settings_dict['OPTIONS'].get('user', settings_dict['USER'])
-        passwd = settings_dict['OPTIONS'].get('passwd', settings_dict['PASSWORD'])
+        password = settings_dict['OPTIONS'].get('password', settings_dict['PASSWORD'])
```

Comment:
I think we can use `passwd` as a fallback, and remove release notes :thinking:
```suggestion
        password = settings_dict['OPTIONS'].get('password', settings_dict['OPTIONS'].get('passwd'), settings_dict['PASSWORD']))
```
What do you think?

Reply:
Ok, I changed the PR and removed the release note.